about.html

```diff
@@ -0,0 +1,135 @@
+                <li class="nav-ul-li">
+                    Sponsor
+                </li>
+                <li class="nav-ul-li">
+                    News
+                </li>
+                <li class="nav-ul-li last">
+                    <button class="crowdfunding">Crowdfunding</button>
+                </li>
+            </ul>
+        </nav>
+        <div class="intro-container">
+            <p class="about-hello">
+                Hello, poets of the world!
+            </p>
+            <h1 class="about-h1">
+                About the Poetic Song Lyrics Project
+            </h1>
+        </div>
+        <p class="about-description">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+            dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+            book.
+        </p>
+        <p class="about-contact">Please contact us per email for any further questions about the website</p>
+        <address class="about-address"><a class="about-email-a" href="mailto:contact@poeticlyrics.com">contact@poeticlyrics.com</a></address>
+    </section>
+    <section class="about-logo">
+        <h2 class="about-h2">
+            Poetic Song Lyrics logo
+        </h2>
+        <p class="logo-desc">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+            dummy text ever since the 1500s.
+        </p>
+        <img src="./images/high-res-logo.png" class="about-logo-img" alt="High res version of logo">
+    </section>
+    <section class="about-past">
+        <h2 class="past-title">
+            See the history of our Project
+        </h2>
+        <p class="past-description">
+            Take a look at our evolution throught the years since our early beginnings.
+        </p>
+        <div class="past-container">
+            <img src="./images/ccsummit2013.png" class="past-imgs" alt="project's own .com domain, in 2013">
+            <img src="./images/ccsummit2011.png" class="past-imgs" alt="project's first great reform, in 2011">
+        </div>
+    </section>
+    <section class="about-partners">
+        <p class="partners-title">
+            Partner
+        </p>
+        <div class="partners-above">
+            <img src="./images/partners/mozilla.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/Google.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/navar.png" class="partner-logo" alt=" logo">
+        </div>
+        <div class="partners-under">
+            <img src="./images/partners/daumkakao.svg" class="partner-logo" alt=" logo">
+            <img src="./images/partners/airbnb.png" class="partner-logo" alt=" logo">
+        </div>
+    </section>
+    <section class="about-footer">
+        <div class="footer-left">
```

Comment:
- [ ] Why don't you use `footer` tag here instead of `section` tag as this is clearly a footer and should have the `footer` tag. This helps to improve the HTML structure of your page and makes your code more [semantic.](https://www.w3schools.com/html/html5_semantic_elements.asp)